--- work/graphviz/backend.py.orig	2018-08-03 01:59:07.623619700 -0500
+++ work/graphviz/backend.py	2018-08-03 04:51:36.313979800 -0500
@@ -61,12 +61,12 @@
 
 PLATFORM = platform.system().lower()
 
-STARTUPINFO = None
-
-if PLATFORM == 'windows':  # pragma: no cover
-    STARTUPINFO = subprocess.STARTUPINFO()
-    STARTUPINFO.dwFlags |= subprocess.STARTF_USESHOWWINDOW
-    STARTUPINFO.wShowWindow = subprocess.SW_HIDE
+def STARTUPINFO():
+    res = subprocess.STARTUPINFO()
+    if PLATFORM == 'windows':  # pragma: no cover
+        res.dwFlags |= subprocess.STARTF_USESHOWWINDOW
+        res.wShowWindow = subprocess.SW_HIDE
+    return res
 
 
 class ExecutableNotFound(RuntimeError):
@@ -123,7 +123,7 @@
 
     with open(os.devnull, 'w') as stderr:
         try:
-            subprocess.check_call(args, startupinfo=STARTUPINFO, stderr=stderr)
+            subprocess.check_call(args, startupinfo=STARTUPINFO(), stderr=stderr)
         except OSError as e:
             if e.errno == errno.ENOENT:
                 raise ExecutableNotFound(args)
@@ -153,7 +153,7 @@
     try:
         proc = subprocess.Popen(args, stdin=subprocess.PIPE,
             stdout=subprocess.PIPE, stderr=subprocess.PIPE,
-            startupinfo=STARTUPINFO)
+            startupinfo=STARTUPINFO())
     except OSError as e:
         if e.errno == errno.ENOENT:
             raise ExecutableNotFound(args)
